src/sentry/incidents/models.py

```diff
@@ -212,6 +215,18 @@ def current_end_date(self):
     def duration(self):
         return self.current_end_date - self.date_started
 
+    def _normalize_before_relocation_import(
```

Comment:
I see the exact same function name and code below. Is it worth to consolidate some of the code and DRY it up if it's used more down the line? I'm pretty neutral on this but wanted to ask

Reply:
Not the same: one refers to `detection_uuid`, the other to `notification uuid`, which are the field names we are regenerating on the respective models. In any case, I generally use a "[rule of 3](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))" before considering DRY.